CaPPMS/Data/GitHubService.cs

```diff
@@ -0,0 +1,127 @@
+﻿using System;
+using System.Threading.Tasks;
+using Octokit;
+namespace CaPPMS.Data
+{
+    public class GitHubService
+    {
+        private GitHubClient gitHubClient = new GitHubClient(new ProductHeaderValue("UMGCApp"));
+
+        public GitHubService()
```

Comment:
GetCredentials?

Reply:
Can you elaborate on this?

Reply:
When leveraging Octokit, that was recommended to use.

Reply:
Just a recommendation for the Method name... but it looks like there was some movement where the comment was meant to be left.